### PR TITLE
Chronos: Test installation options(forecaster)

### PR DESCRIPTION
**Pull request by liangs6212**
## Description

add later

### 1. Why the change?

add later
related issue: #4791

### 2. User API changes

No changed

### 3. Summary of the change 

add later

### 4. How to test?

- [ ] Unit test
- [ ]  Github action

### 5. Possible Usage

only torch: pytest -v -m "torch and not distributed and not all and not automl" <your_path>/file.py
torch + distributed: pytest -v -m "torch and not automl and not all" <your_path>/file.py
torch + automl: pytest -v -m "torch and not distributed and not all" <you_path>/file.py
all:  Do not use -m
all but without other deps: pytest -v -m "all" <your_path>/file.py

### CODE REVIEW

**Review by TheaperDeng — APPROVED** [review]
merge after #5165

**Review by TheaperDeng — COMMENTED** [review]
please fix conflict first

**Comment by liangs6212** [comment]
http://10.112.231.51:18889/view/BigDL-PR-Validation/job/BigDL-Chronos-PR-Validation/811/

**Comment by TheaperDeng** [comment]
pls rebase and we just need to make sure the github actions can be passed

**Comment by liangs6212** [comment]
> pls rebase and we just need to make sure the github actions can be passed

Done.

File: python/chronos/dev/test/run-installation-options.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#
+
+cd "`dirname $0`"
+cd ../..
+
+export PYSPARK_PYTHON=python
+export PYSPARK_DRIVER_PYTHON=python
+if [ -z "${OMP_NUM_THREADS}" ]; then
+    export OMP_NUM_THREADS=1
+fi
```
[the remark this answers is not in the document]
Comment (liangs6212) [reply python/chronos/dev/test/run-installation-options.sh:26]: I'm not sure, because it's copied from another script.

**Pull request merged**